clang/test/dpct/cmake_migration/case_035/input.cmake

```diff
@@ -0,0 +1,11 @@
+set(CUDA_64_BIT_DEVICE_CODE ON)
+
+set(CUDA_64_BIT_DEVICE_CODE OFF)
+
+if(CUDA_64_BIT_DEVICE_CODE)
```

Comment:
This case?

* `if (NOT CUDA_64_BIT_DEVICE_CODE)`

Reply:
Done.